src/utility/watchdog/Watchdog.cpp

```diff
@@ -56,3 +73,31 @@ void wifi_nina_feed_watchdog()
   samd_watchdog_reset();
 }
 #endif /* ARDUINO_ARCH_SAMD */
+
+#ifdef ARDUINO_ARCH_MBED
+void mbed_watchdog_enable()
+{
+  watchdog_config_t cfg;
+#if defined(ARDUINO_PORTENTA_H7_M7) || defined(ARDUINO_PORTENTA_H7_M)
```

Comment:
`ARDUINO_PORTENTA_H7_M` should be `ARDUINO_PORTENTA_H7_M4` :wink:

Reply:
Good catch!